cmake/Modules/FindAWSSDK_EP.cmake

```diff
@@ -100,6 +100,8 @@ if (NOT AWSSDK_FOUND)
       DOWNLOAD_NAME ep_awssdk.zip
       URL "https://github.com/aws/aws-sdk-cpp/archive/1.8.84.zip"
       URL_HASH SHA1=e32a53a01c75ca7fdfe9feed9c5bbcedd98708e3
+      PATCH_COMMAND
+        patch -N -p1 < ${TILEDB_CMAKE_INPUTS_DIR}/patches/ep_awssdk/awsccommon.patch
```

Comment:
`patch` won't be available in a typical windows/MSVC build environment, so we need to use `git apply` here, as in the `FindAzure` and `FindCapnp` files.

(we do use `patch` in the GCS build, but that is because we don't build GCS on windows yet)